src/components/BlogPost.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from "react";
+import Markdown from 'markdown-to-jsx';
+
+const fm = require('front-matter');
```

Comment:
You should be able to use ESM import syntax here

So like:

`import fm from 'front-matter'`